zanata-war/src/main/java/org/zanata/exception/RequestExistException.java

```diff
@@ -0,0 +1,11 @@
+package org.zanata.exception;
+
+/**
+ * @author Alex Eng <a href="aeng@redhat.com">aeng@redhat.com</a>
+ */
+public class RequestExistException extends ZanataServiceException {
```

Comment:
I can't tell what this is (javadoc please), but RequestExistsException might be better.